### ISSUE TITLE
Age 1 - Shift clicking ore clusters gives rocks instead of the corresponding ore.

### ISSUE THREAD

**Issue by Osmiumite** [Darkosto/SevTech-Ages#1166]
**Storytime/Bug**
Shortly after entering age 1, and working my way up the advancement tree to "Locate and Mine Coal," "Error 404: Heart Not Found," and "Get to The Copper!"(Nice job on that one (: ), I went to search for some ore clusters in my already-genned area from the tutorial stage(Fresh world, single player). I found some coal (and copper) and shift-clicked them, only to get rocks; However, when I break them by hand/with tools I recieve the correct ore sample. I haven't tested in non-explored territory yet.

I believe in the "Prospecting" book given to you after crafting a plume, it says you can shift-click to pick up the cluster. 

**TL;DR**
Shift clicking copper and coal samples on the ground gives rocks in age 1.

**Solution-ish?**
Make the clusters unable to be shiftclicked?

**Steps to Reproduce (for bugs)**
>explore a good few chunks that have ore samples(disguised as rocks)
>Enter Age 1
>shift-click cluster
>profit!.. wait what?

## Your Environment

* Modpack Version: 3.03
* Java Version: 1.8.0 144 64bit
* Launcher Used: Twitch/Jar preset
* Resourcepack Enabled?: Nope
* Optifine Installed?: No, only the default mods in the pack.
